R/corenet_build_OS.R

```diff
@@ -65,7 +65,17 @@ corenet_build_OS = function(os_scotland, osm_scotland, region_names,cities_regio
                 geometry = st_line_merge(st_combine(st_union(geometry)))
               )
 
-            orcp_city_boundary$road_function = "Off Road/Detached Cycle Track/Path"
+            p_90 = quantile(orcp_city_boundary$all_fastest_bicycle_go_dutch, 0.90, na.rm = TRUE)
```

Comment:
Looks good. We can adjust this based on feedback. What do you think of making this an argument in the function that the users (us) can change easily?

Reply:
Also wondering if you think values above or below 0.9 are more likely to generate better results. Happy with this as a starter for 10.